### DISCUSSION TITLE
How to change device in GC?

### DISCUSSION THREAD

**Discussion by tonylchang** [philosowaffle/peloton-to-garmin#217] [Q&A] [answered]
Hello!

I am an existing Garmin user who picked up a Peloton. I was able to get the Windows build running with minimum effort, but noticed that the device in Garmin Connect is different on workouts that P2G uploaded. The P2G device on my end is "Forerunner 945" running software version 5.0. Is there a way to change this? 

Thanks for the great work.

Tony

**Comment by Vicore1** [comment, marked as answer]
Yup! Take a look at this information in the wiki
https://philosowaffle.github.io/peloton-to-garmin/configuration/json.html#custom-device-info

**Comment by tonylchang** [comment]
Thanks for the quick response!







---- On Thu, 03 Feb 2022 11:44:37 -0800 Vicore1 ***@***.***> wrote ----





Yup! Take a look at this information in the wiki
 https://philosowaffle.github.io/peloton-to-garmin/configuration/json.html#custom-device-info

**Comment by NateEverden** [comment]
This doesn't seem to completely work for me.  It's not showing the minor version (20.00 vs 20.30).

Here is what I have in my device file
`<Creator>
  <Name>fenix 6 Pro Solar</Name>
  <UnitId>9999999</UnitId>
  <ProductID>3290</ProductID>
  <Version>
    <VersionMajor>20</VersionMajor>
    <VersionMinor>30</VersionMinor>
    <BuildMajor>0</BuildMajor>
    <BuildMinor>0</BuildMinor>
  </Version>
</Creator>`


FYI - I changed my unit ID.  The activities I upload show version 20.00 when it should show 20.30.

**Reply by philosowaffle** [reply]
There is already an open issue for this that can be followed here: #218 

To my knowledge, whether the minor version displays correctly or not has no impact on functionality. Purely cosmetic.

**Reply by NateEverden** [reply]
thanks!
